tests/buildprocfile_test.go

```diff
@@ -40,7 +40,8 @@ var _ = Describe("deis builds procfile", func() {
 				apps.Destroy(user, app)
 			})
 
-			Specify("that user create a new build of that app with a different procfile", func() {
+			// TODO(smothiki): somehow determine why this test is flaky in jenkins after merging deployments
```

Comment:
Could you reference the GH issue here? https://github.com/deis/workflow-e2e/issues/299